pkg/artifactory/resource/security/resource_artifactory_keypair_test.go

```diff
@@ -276,6 +279,12 @@ func TestAccKeyPairGPG(t *testing.T) {
 		=fot9
 		-----END PGP PUBLIC KEY BLOCK-----
 		EOF
+			lifecycle {
+			ignore_changes = [
+					private_key,
+					passphrase,
```

Comment:
These should not be necessary since `keyPairPacker()` should ignore `private_key`. We should add `passphrase` to the ignore list.

Reply:
Ok, removed and added `passphrase` to the ignore list.